javadsl/src/main/java/com/example/scdfdsl/ScdfdslApplication.java

```diff
@@ -52,8 +64,8 @@ public StreamApplication sink() {
 
 	@Override
 	public void run(ApplicationArguments applicationArguments) throws Exception {
-		DataFlowOperations dataFlowOperations = createDataFlowOperations(applicationArguments);
-
+//		dataFlowOperations.appRegistryOperations().importFromResource(
```

Comment:
Should decide if this line is in our out.

Reply:
That was my original question in the comments